content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md

```diff
@@ -0,0 +1,199 @@
+---
+title: ネームスペースのデフォルトのメモリ要求と制限を設定する
+content_type: task
+weight: 10
+description: >-
+  ネームスペースのデフォルトのメモリリソース制限を定義して、そのネームスペース内のすべての新しいPodにメモリリソース制限が設定されるようにします。
+---
+
+<!-- overview -->
+
+このページでは、{{< glossary_tooltip text="ネームスペース" term_id="namespace" >}}のデフォルトのメモリ要求と制限を設定する方法を説明します。
+
+Kubernetesクラスターはネームスペースに分割することができます。デフォルトのメモリ[制限](ja/docs/concepts/configuration/manage-resources-containers/#%E8%A6%81%E6%B1%82%E3%81%A8%E5%88%B6%E9%99%90)を持つネームスペースがあり、独自のメモリ制限を指定しないコンテナでPodを作成しようとすると、{{< glossary_tooltip text="コントロールプレーン" term_id="control-plane" >}}はそのコンテナにデフォルトのメモリ制限を割り当てます。
```

Comment:
```suggestion
Kubernetesクラスターはネームスペースに分割することができます。デフォルトのメモリ[制限](/ja/docs/concepts/configuration/manage-resources-containers/#%E8%A6%81%E6%B1%82%E3%81%A8%E5%88%B6%E9%99%90)を持つネームスペースがあり、独自のメモリ制限を指定しないコンテナでPodを作成しようとすると、{{< glossary_tooltip text="コントロールプレーン" term_id="control-plane" >}}はそのコンテナにデフォルトのメモリ制限を割り当てます。
```

and can you add an English ID for the link target page, and use it?

Reply:
> and can you add an English ID for the link target page, and use it?

I created the PR https://github.com/kubernetes/website/pull/33213. Please review.